stash/src/server.ts

```diff
@@ -23,9 +23,7 @@ const server = app.listen(app.get('port'), async () => {
 
   await networkService.initService()
 
-  if (process.env.APP_ENV !== 'rollup-dev') {
```

Comment:
Since we have /xcm endpoints in app.ts it made no sense to not initialize data for it.